pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -54,17 +62,17 @@ func (o *OssmInstaller) processManifests() error {
 	// TODO warn when file is not present instead of throwing an error
 	// IMPORTANT: Order of locations from where we load manifests/templates to process is significant
 	err := o.loadManifestsFrom(
-		path.Join("control-plane", "base"),
-		path.Join("control-plane", "filters"),
-		path.Join("control-plane", "oauth"),
-		path.Join("control-plane", "smm.tmpl"),
-		path.Join("control-plane", "namespace.patch.tmpl"),
-
-		path.Join("authorino", "namespace.tmpl"),
-		path.Join("authorino", "smm.tmpl"),
-		path.Join("authorino", "base"),
-		path.Join("authorino", "rbac"),
-		path.Join("authorino", "mesh-authz-ext-provider.patch.tmpl"),
+		path.Join("templates", "control-plane", "base"),
+		path.Join("templates", "control-plane", "filters"),
+		path.Join("templates", "control-plane", "oauth"),
+		path.Join("templates", "control-plane", "smm.tmpl"),
+		path.Join("templates", "control-plane", "namespace.patch.tmpl"),
+
+		path.Join("templates", "authorino", "namespace.tmpl"),
+		path.Join("templates", "authorino", "auth-smm.tmpl"),
+		path.Join("templates", "authorino", "base"),
+		path.Join("templates", "authorino", "rbac"),
+		path.Join("templates", "authorino", "mesh-authz-ext-provider.patch.tmpl"),
```

Comment:
We could make two const here and reuse instead:

```
ControlPlaneDir = "templates/control-plane"
AuthDir = "templates/authorino"
```

Reply:
[1ce2892](https://github.com/maistra/opendatahub-operator/pull/5/commits/1ce2892d25dc3fa0bf60fe4b742672227c458560) added here

Reply:
Technically you defined variables. Make them `const` instead. Cannot comment on the line from GitHub mobile for whatever reason :)

Reply:
done now here! https://github.com/maistra/opendatahub-operator/commit/b0251b064975d493b49e67f619ab0db0994c90bb

Reply:
I don't see this commit on the list here...